packages/vm/scripts/benchmarks/mainnetBlocks.ts

```diff
@@ -70,6 +70,7 @@ export async function getPreState(pre: {
   await state.checkpoint()
   for (const k in pre) {
     const kBuf = toBuffer(k)
+    const addr = new Address(kBuf)
```

Comment:
This throws if `kBuf` is not 20 bytes.

Reply:
i'm not understanding your concern here

Reply:
Yeah this actually isn't really a problem, we can expect that test addresses are padded the right way.